### PR TITLE
Add Irish translation

### PR DESCRIPTION
**Pull request by yrest**

### CODE REVIEW

File: samples/HelloWorld/GreetingsManagement.al [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (StanislawStempin) [inline samples/HelloWorld/GreetingsManagement.al]: Please remove this empty line

Comment (yrest) [inline samples/HelloWorld/GreetingsManagement.al]: Empty line has been removed

Comment (StanislawStempin) [inline samples/HelloWorld/GreetingsManagement.al]: It is still there - line 1

Comment (yrest) [inline samples/HelloWorld/GreetingsManagement.al]: Sorry, didn't see that on line 1, removed now

Comment (yrest) [inline samples/HelloWorld/GreetingsManagement.al]: Oh, I understand now! This whole conversation is related to that one line!

**Pull request merged**